modules/javafx.graphics/src/main/java/com/sun/glass/ui/gtk/GtkApplication.java

```diff
@@ -176,15 +158,14 @@ private static float getFloat(String propname, float defval, String description)
 
         if (gtkVersion == 2 && !gtk2WarningIssued) {
             System.err.println(GTK2_SPECIFIED_WARNING);
-            System.err.println(GTK2_DEPRECATION_WARNING);
+            System.err.println(GTK2_REMOVED_WARNING);
             gtk2WarningIssued = true;
+            gtkVersion = 3;
         }
 
         @SuppressWarnings("removal")
         boolean gtkVersionVerbose =
-                AccessController.doPrivileged((PrivilegedAction<Boolean>) () -> {
-            return Boolean.getBoolean("jdk.gtk.verbose");
-        });
+                AccessController.doPrivileged((PrivilegedAction<Boolean>) () -> Boolean.getBoolean("jdk.gtk.verbose"));
```

Comment:
Minor: Since you are not otherwise touching this block, it would be cleaner to revert this reformatting change.

Reply:
I have a thing for lambdas.